lib/lrama/lexer.rb

```diff
@@ -18,8 +20,8 @@ def to_s
         "#{super} line: #{line}, column: #{column}"
       end
 
-      @i = 0
-      @types = []
+      instance_variable_set :@i, 0
```

Comment:
[note] steep can not detect a type of instance variable for original codes, even so rbs file has [declarations for them](https://github.com/ruby/lrama/pull/33/files#diff-fd876d31684b261c8afa3dc51c8f0ad63e8a481babf3564cab0ecbb565a7e1f7R44-R45). It might the limitation for Struct with block (?).

```
lib/lrama/lexer.rb:23:6: [error] Cannot find the declaration of instance variable: `@i`
│ Diagnostic ID: Ruby::UnknownInstanceVariable
│
└       @i = 0
        ~~

lib/lrama/lexer.rb:24:6: [error] Cannot find the declaration of instance variable: `@types`
│ Diagnostic ID: Ruby::UnknownInstanceVariable
│
└       @types = []
        ~~~~~~

Detected 2 problems from 1 file
```